flow/Makefile

```diff
@@ -279,7 +279,18 @@ YOSYS_CMD               ?= $(abspath $(FLOW_HOME)/../tools/install/yosys/bin/yos
 
 KLAYOUT_CMD             ?= $(shell command -v klayout)
 
-KLAYOUT_FOUND            = $(if $(KLAYOUT_CMD),,$(error KLayout not found in PATH))
+MIN_VERSION = 0.28.11
```

Comment:
@vvbandeira in order to accept this will we need to update klayout on the OR cluster (and maybe the jenkins image?)
```
klayout -v
KLayout 0.28.9
```

Reply:
Apologies for the delay. I have been busy with other time-sensitive tasks and now I am resolving a critical issue with some of our servers. As soon as this is resolved, I will get back to you on this issue -- ETA 24-36h.

Reply:
@oharboe I had done the work but forgot to restart the tests 😅
Looks like there are some issues with the path of some of the files. All the designs are failing with similar messages.

```
ERROR: RuntimeError: Unable to open file: /tmp/workspace/public_tests_all-pr_PR-1364-head/flow/results/asap7/uart/base//platforms/asap7/lef/asap7sc7p5t_28_R_1x_220121a.lef (errno=2) in Layout.read
  ./util/def2stream.py:113
```